refactor: convert proxy type page blockquotes to admonitions (#2658)
apify/apify-docs: sources/platform/proxy/datacenter_proxy.md

You can find your proxy password on the [Proxy page](https://console.apify.com/proxy) of Apify Console.

:::note Username field

> The `username` field is _not_ your Apify username. Instead, you specify proxy settings (for example, `groups-BUYPROXIES94952`, `session-123`). Use `auto` for default settings.

Reviewer: I'm confused. Does it want to say that you use `username` field to specify proxy settings?

Author: Basically `username` despite its unfortunate name has nothing to do with actual username... (see [here](https://docs.apify.com/platform/proxy/usage#external-connection))

Perhaps rephrasing to lead with what it is actually instead of what isn't will be more clear

```suggestion
Use the `username` field to specify your proxy settings (for example, `groups-BUYPROXIES94952`, `session-123`), or set it to `auto` for default settings. It is not your Apify username.
```

And same rewrite on SERP page.

Reviewer: Yup, the new version explains it in much more clear way

Author: Will rewrite both of those in that way then 💪
